scania/corporate-ui-site
# Feature/image template

**Describe pull-request**  
Add image template for one column and multiple columns


**How to test**  
Check Figma > Page-Template > Image
Check few examples on different pages (In CMS the image need to have correct size to make it works)
Try Components > Buttons > Style 
Components > Dividers > Usage

**Screenshots**  
![image](https://user-images.githubusercontent.com/1199101/101600266-5c1e5b80-39fb-11eb-8089-92964e986368.png)


**Additional context**  
_Add any other context about the pull-request here._
